web/frontend/src/pages/ballot/components/Text.tsx

```diff
@@ -8,11 +8,14 @@ type TextProps = {
   text: TextQuestion;
   answers: Answers;
   setAnswers: (answers: Answers) => void;
+  language: string;
 };
 
-const Text: FC<TextProps> = ({ text, answers, setAnswers }) => {
+const Text: FC<TextProps> = ({ text, answers, setAnswers, language }) => {
   const { t } = useTranslation();
-  const [charCounts, setCharCounts] = useState(new Array<number>(text.Choices.length).fill(0));
+  const [charCounts, setCharCounts] = useState(
+    new Array<number>(text.ChoicesMap.get('en').length).fill(0)
```

Comment:
it's better to let 'en' because if someone didn't fill the french language but is displaying the french part the choicesmap.get('fr') will be empty and it will cause an error . But since, the english is mandatory we can't have this problem

Reply:
But if there is indeed the french available, how can it get displayed ?